modules/nlu/src/test/types.ts

```diff
@@ -0,0 +1,25 @@
+import { IO } from 'botpress/sdk'
+
+/**
+ * @description extending IO.EventUnderstanding as is is necessary in this context (nlu/backend/election) as the Typescript compiler doesn't allow me to import botpress/sdk in a test file
+ * @todo fix Typescript configuration file to allow botpres/sdk import in test files
+ */
+export type EventUnderstanding = IO.EventUnderstanding
+
+export function fakeEventUnderstanding(x?: Partial<EventUnderstanding>): EventUnderstanding {
```

Comment:
**PP**

This is neither a fake or a mock, it's just an event understanding with default values.

Fake and mocks are what we call "_test doubles_" (this also includes, stubs and dummies). A test double usually reimplement the behavior of a method for the purpose of a unit test, i.e. a test that asserts the behavior of a single component / module / package / class / function / part_of_code.

Again: **I really don't mind, it's just naming**